libs/utils/utils.js

```diff
@@ -1598,6 +1598,11 @@ async function processSection(section, config, isDoc) {
 
 export async function loadArea(area = document) {
   const isDoc = area === document;
+  const isPageLoadOk = document.getElementById('page-load-ok-milo') !== null;
+
+  if (isDoc && isPageLoadOk) {
+    return;
+  }
```

Comment:
Since this isn't re-used and `utils` is (or at least should be) a place of brevity, how about
```suggestion
  if (isDoc && !!document.getElementById('page-load-ok-milo')) return;
```

Reply:
@overmyheadandbody resolved in eca2049